cockatrice/src/player.cpp

```diff
@@ -733,7 +733,7 @@ void Player::setShortcutsActive()
     aPlay->setShortcuts(settingsCache->shortcuts().getShortcut("Player/aPlay"));
     aTap->setShortcuts(settingsCache->shortcuts().getShortcut("Player/aTap"));
     aDoesntUntap->setShortcuts(settingsCache->shortcuts().getShortcut("Player/aDoesntUntap"));
-    aFlip->setShortcuts(settingsCache->shortcuts().getShortcut("Player/aFlip"));
+    aTurnOver->setShortcuts(settingsCache->shortcuts().getShortcut("Player/aTurnOver"));
```

Comment:
This string looks weird to me. What is exactly used for?

Reply:
Settings are saved to an ini file; that's the "key" used to identify that specific setting in the file, eg:.
```
[Custom]
...
Player\aDrawCards=Ctrl+E
```